app/channels/room_channel.rb

```diff
@@ -0,0 +1,26 @@
+class RoomChannel < ApplicationCable::Channel
+  def subscribed
+    reject if params[:room_id].blank?
+    stream_from("ChatRoom-#{params[:room_id]}")
+  end
+
+  def send_message(data)
+    sender    = current_user
+    room_id   = params[:room_id]
+    message   = data['message']
+    raise 'No room_id!' if room_id.blank?
+
+    convo = get_convo(room_id)
+    raise 'No message!' if message.blank?
+
+    Message.create!(chat: convo, user: sender, text: message)
+    payload = {
+      room_id: room_id, content: message, sender: sender, participants: convo.users
+    }
+    ActionCable.server.broadcast("ChatRoom-#{room_id}", payload)
+  end
+
+  def get_convo(room_code)
```

Comment:
Maybe you can rename this to:
```
def get_chat(room_code) 
OR 
def get_conversation(room_code)
```

Also I'm not sure if this method is only used in this class or is also used by the frontend. In case it is only used within this class, you can delcare it as private.